src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final double MAX_SIZE = 10.0;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomValue = RANDOM.nextInt(5);
+        if (randomValue == 0) {
+            return new Square(colorSupplier.getRandomColor(), RANDOM.nextDouble() * MAX_SIZE);
```

Comment:
```suggestion
            return new Square(randomColor, RANDOM.nextDouble() * MAX_SIZE);
```

Check each type of figure

create separate method for getting random side -> `RANDOM.nextDouble() * MAX_SIZE` and use it for each type of figure